source/source-connector/configuration-properties/output-format.txt

```diff
@@ -56,6 +56,14 @@ Settings
        | **Default**: ``json``
        | **Accepted Values**: ``bson``, ``json``, ``schema``
 
+       .. note:: Protobuf Output Format Support
+
+          The source connector supports Protobuf as an
+          output format if you specify ``output.format.value=schema``
+          and :ref:`configure your connector
```

Comment:
Suggestion:
I think this is how to enable it rather than a condition for support (currently phrased as "X supports Y if you specify Z"). You can also omit "source" to identify the connector since this is the page for source connector config. Similarly, this is the output.format.value setting, so you don't need to mention what the name part of the setting looks like unless it is a syntax example.
Also, from the Confluent page link, it looks like the reader needs to install the converter and then configure it whereas the text only mentions the latter.

 E.g.
```
  The connector supports the Protobuf output data format. You can enable it
  by specifying the ``schema`` option and installing and configuring the 
  `Kafka Connect Protobuf Converter <https://www.confluent.io/hub/confluentinc/kafka-connect-protobuf-converter>`__.
```